careers/findingjobs.md

```diff
@@ -17,6 +17,14 @@ on your own. Some good places to search:
     fantastic companies that offer internships.
 -   [Work In Tech](https://www1.communitech.ca/jobs) has an extensive list of job opportunities, though most of
     them are not internships.
+-   [Pitt-CSC](https://github.com/pittcsc) has a giant open source repo that tracks open CS internships for a lot of companies. The organization also have a repo for new grad roles.
```

Comment:
change `have` to has. There is also no new grad repo anymore, the maintained fork can be found https://github.com/coderQuad/New-Grad-Positions-2023